src/backend_common.cc

```diff
@@ -673,6 +673,23 @@ ReadTextFile(const std::string& path, std::string* contents)
   return nullptr;  // success
 }
 
+TRITONSERVER_Error*
+ReadBinaryFile(const std::string& path, std::vector<uint8_t>& contents)
```

Comment:
This seems to be the same as ReadTextFile except the result ends up in a vector instead of a string. So we can have the function but prefer that it is still called ReadTextFile. I also notice that the armnn API has a create that takes an istream, why not just create the std::ifstream and call that one?

Reply:
Yeah I agree. That's the better way to do things. I'll just leave that logic in the ArmNN backend.